caluma/caluma_core/types.py

```diff
@@ -85,14 +85,14 @@ def resolve_connection(cls, connection, args, iterable, max_limit=None):
         else:  # pragma: no cover
             _len = len(iterable)
 
-        # If after is higher than list_length, connection_from_list_slice
+        # If after is higher than list_length, connection_from_array_slice
         # would try to do a negative slicing which makes django throw an
         # AssertionError
         after = min(get_offset_with_default(args.get("after"), -1) + 1, _len)
         if max_limit is not None and "first" not in args:  # pragma: no cover
             args["first"] = max_limit
 
-        connection = connection_from_list_slice(
+        connection = connection_from_array_slice(
```

Comment:
and here the kwargs-keys containing `list` -> `array`

Reply:
done